velox/common/encode/Base64.cpp

```diff
@@ -249,191 +254,198 @@ std::string Base64::encode(folly::StringPiece text) {
 }
 
 // static
-std::string Base64::encode(const char* data, size_t len) {
-  return encode(folly::StringPiece(data, len));
+std::string Base64::encode(const char* input, size_t inputSize) {
+  return encode(folly::StringPiece(input, inputSize));
 }
 
 namespace {
 
 /**
- * this is a quick and dirty iterator implementation for an IOBuf so that the
- * template that uses iterators can work on IOBuf chains.  It only implements
- * postfix increment because that is all the algorithm needs, and it is a noop
- * since the read<>() function already incremented the cursor.
+ * This is a quick and simple iterator implementation for an IOBuf so that the
```

Comment:
Let's fix this by using `//`.
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md